atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -82,8 +82,7 @@ def _get_registries(self):
         Return a list of registries that are exposed to user
         If pulp registries are used, only those will be returned
         """
-        return (self.workflow.push_conf.pulp_registries or
-                self.workflow.push_conf.all_registries)
+        return (self.workflow.push_conf.all_registries)
```

Comment:
Method's docstring is wrong. "Return a list of all the registries this build updated"?
Unnecessary parentheses.